docs/cypress/e2e/links.cy.ts

```diff
@@ -0,0 +1,136 @@
+import { VALIDATED_LINKS } from '../data/validatedLinks';
+import { REQUEST_GET_LINKS } from '../data/requestGetLinks';
+import { BASE_URL } from '../data/constants';
+
+let allLinks: string[] = [];
+const numberOfLinks = 119;
+const requestedLinks: Set<string> = new Set();
+
+before(() => {
+  cy.task('readSitemapLinks').then((links: string[]) => {
+    allLinks = allLinks.concat(links);
+  });
+});
+
+describe('Local Sitemap', () => {
+  it('should have 119 links', () => {
+    expect(allLinks.length).to.eq(numberOfLinks);
+  });
+});
+
+describe(`All pages on Sitemap`, () => {
+  for (let i = 0; i < numberOfLinks; i++) {
+    it(`all links on page ${i} should work`, () => {
```

Comment:
If it works, it would look cleaner, but I tested it and it doesn't work because `allLinks` is generated at `before`, when `it.each(allLinks)` , it'd be `undefined`.

Or we can use something like ` it.each(new Array(numberOfLinks).fill(1))`, but I'm not sure how we can get the index 🤔

Reply:
NVM, I figured it out. We just need to use [`cypress-each`](https://github.com/bahmutov/cypress-each#repeat-the-test-n-times).

Updated the PR.